components/atoms/loading/ChatLoading.tsx

```diff
@@ -0,0 +1,21 @@
+import React, {FC} from "react";
+
+
+export const ChatLoading:FC=()=>{
```

Comment:
Abstract to a `Spinner` component with just the SVG, then have `className` get passed in and `cn`'d on